Src/BootCamp.Chapter/Program.cs

```diff
@@ -7,6 +7,55 @@ class Program
         static void Main(string[] args)
         {
             Console.WriteLine("Hello World!");
+            Console.WriteLine("Hello World!");
+
+            //Get Name
+            Console.Write("What is your name? ");
+            var myName = Console.ReadLine();
+            Console.WriteLine("Hello, " + myName + "!");
+            //Get Age
+            Console.Write("How old are you? ");
+            var myAge = Console.ReadLine();
+            //Get Weight
+            Console.Write("How much do you weigh? ");
+            double myWeight = Convert.ToDouble(Console.ReadLine());
+            //Get Height
+            Console.Write("What is your height? ");
+            double myHeight = Convert.ToDouble(Console.ReadLine());
+            //User Info
+            Console.WriteLine(myName + " is " + myAge + " years old, their weight is " + myWeight + "kg and their height is " + myHeight + " cm.");
+
```

Comment:
Two blank lines are never really needed to separate anything- 1 is enough. But it's good that you separate unrelated things.